Doc update

pedro-garridor opened the pull request:
### Is there an existing PR for this?
- [x] I have searched for existing PRs

### Types of changes:
- [x] Bug fix (change which fixes an identified problem)
- [ ] "New" feature (change which adds or improves existing functionality)

### Description
Please include a summary of the changes and the related issue(s). List any dependencies that are required for this change, if any.

Update pipeline doc to include new info and latest changes derived from a new implemented variant caller (GATK Mutect2).

Fixes [#75](https://github.com/CIBERER/GdTBioinfo-nf-mosaicism/issues/75#issue-2351451979)

### Checklist:
Please verify that you have completed the following:
- [x] Guidelines adherence: my changes follow the [guidelines](implementation_guidelines.md) of this project.
- [x] Documentation update: I have made corresponding changes to the documentation, if necessary.
- [ ] Test inclusion: I have added tests that prove my code is effective and works.
- [ ] New and existing unit tests pass locally with my changes.

pedro-garridor committed:
Update doc

Update doc including mutect2 part and new merged results.

pedro-garridor committed:
Update username and pipeline name on README

Change `CIBERER-pipelines/mosaicism` for `CIBERER/GdTBioinfo-nf-mosaicism`.

pedro-garridor committed:
Update profile_test.md5

Update `.tests/profile_test.md5` with md5 info for new testdata output.

yocra3 commented:
Hola @pedro-garridor, 
Antes de hacer merge, ¿Puedes revisar este documento: https://nf-co.re/docs/guidelines/external_use? Aquí explica como hacer pipelines con el template de nf-core cuando no van a ir a nf-core. ¿Puedes hacer los cambios que proponen aquí antes de hacer el PR?

yocra3 reviewed (changes requested):
Antes de hacer merge, ¿Puedes revisar este documento: https://nf-co.re/docs/guidelines/external_use? Aquí explica como hacer pipelines con el template de nf-core cuando no van a ir a nf-core. ¿Puedes hacer los cambios que proponen aquí antes de hacer el PR?

pedro-garridor committed:
Compliance with nf-core for external use

Changes to comply with [nf-core guidelines to use nf-core resources outside nf-core](https://nf-co.re/docs/guidelines/external_use).

pedro-garridor committed:
Update README

Show CIBERER logo

pedro-garridor committed:
Update README

Correct CIBERER logo path
